src/isomorphic/modern/class/__tests__/ReactClassEquivalence-test.js

```diff
@@ -32,7 +32,7 @@ describe('ReactClassEquivalence', function() {
 function runJest(testFile) {
   var cwd = process.cwd();
   var jestBin = path.resolve('node_modules', '.bin', 'jest');
-  var setupFile = path.resolve(__dirname, 'setupSpecEquivalenceReporter.js');
+  var setupFile = path.resolve(__dirname, 'setupSpecEquivalenceReporter.js.setup');
```

Comment:
We can either
- move the `setupSpecEquivalenceReporter.js` file outside of `__tests__` folder or
- hardcode that file name in the package.json config

I think the current solution is better than those two others. Let me know if you have a better idea